assets/_scss/_navigation.scss

```
@@ -280,13 +280,18 @@ Overlay : allways ***********************
 			}
 		}
 	}
-
-	// ナビ背景色がない場合に、コアがモーダルの背景に白を指定してくるため、
-	// フッターなど親要素の背景が暗くて文字色を白に指定している場合に、
-	// モーダル背景白 / 文字色白 になってしまう不具合があるため、モーダル背景を上書き
-	// ※ オーバーレイテンプレートの場合はそちらで直接色指定すればいいのでこちらでは指定しない
-	.has-bg-dark-background-color &.wp-block-navigation__responsive-container.is-menu-open:not(.disable-default-overlay) {
-		background-color: var(--wp--preset--color--bg-dark);
+	// デフォルトモーダルでの色指定 ///////////
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)